src/frontend/test_runner/tests/testdata/tpch.yaml

```diff
@@ -1661,48 +1661,56 @@
                 LogicalFilter { predicate: Like($7, '%Customer%Complaints%':Varchar) }
                   LogicalScan { table: supplier, columns: [_row_id, s_suppkey, s_name, s_address, s_nationkey, s_phone, s_acctbal, s_comment] }
   optimized_logical_plan: |
-    LogicalAgg { group_key: [0, 1, 2], agg_calls: [count(distinct $3)] }
-      LogicalProject { exprs: [$1, $2, $3, $0] }
-        LogicalJoin { type: LeftAnti, on: ($0 = $4), output_indices: all }
-          LogicalJoin { type: Inner, on: ($2 = $0), output_indices: [1, 3, 4, 5] }
-            LogicalScan { table: partsupp, columns: [ps_partkey, ps_suppkey] }
-            LogicalScan { table: part, output_columns: [p_partkey, p_brand, p_type, p_size], required_columns: [$1:p_partkey, $4:p_brand, $5:p_type, $6:p_size], predicate: ($4 <> 'Brand#45':Varchar) AND Not(Like($5, 'SMALL PLATED%':Varchar)) AND In($6, 19:Int32, 17:Int32, 16:Int32, 23:Int32, 10:Int32, 4:Int32, 38:Int32, 11:Int32) }
-          LogicalScan { table: supplier, output_columns: [s_suppkey], required_columns: [$1:s_suppkey, $7:s_comment], predicate: Like($7, '%Customer%Complaints%':Varchar) }
+    LogicalAgg { group_key: [0, 1, 2], agg_calls: [count($3) filter(($4 = 0:Int64))] }
+      LogicalAgg { group_key: [0, 1, 2, 3, 4], agg_calls: [] }
+        LogicalExpand { column_subsets: [[0, 1, 2, 3]] }
```

Comment:
Maybe open an issue to simplify / remove unnecessary expand.

Reply:
#3806